### DISCUSSION TITLE
Reducing download time

### DISCUSSION THREAD

**Discussion by Klaudioz** [AbdBarho/stable-diffusion-webui-docker#212] [General]
Hello, is any way to choose to avoid downloading certain files?

I think it can be done just by editing the `./services/download/links.txt`, but probably doing it on the `docker-compose.yml` should be a better practice.

**Comment by AbdBarho** [comment]
@Klaudioz I do agree with enabling features over `docker-compose.yml` is a better practice, however, some UIs expect the files to be there and will crash if they don't find them.

I know that the v1.4 model https://github.com/AbdBarho/stable-diffusion-webui-docker/blob/37feff58bb715dcb239f7fceb87112e66251feef/services/download/links.txt#L9 is not necessary anymore and can be removed, 

Maybe the LDSR model as well, but no guarantees...

**Reply by Klaudioz** [reply]
Thanks for your answer. I didn't know that almost everything is needed. Also I'm going to provide my own model (Dreambooth) so probably I can skip an extra one.
